dask_cuda/dask_cuda_worker.py

```diff
@@ -113,6 +114,12 @@
     "'auto', or zero for no memory management "
     "(i.e., allow full device memory usage).",
 )
+@click.option(
+    "--rmm-pool-size",
+    default=None,
+    help="Initialize each worker with an RMM pool of the given size. "
+    "This can be an integer (bytes) or string (like 5GB or 5000M)."
```

Comment:
Should we say something about it being disabled by default?

Reply:
Done.